src/chapters/tableau-intro/studio.rst

```diff
@@ -1,2 +1,33 @@
 Studio: Tableau Part 1 - Introduction to Visualization 
-======================================================
\ No newline at end of file
+======================================================
+
+Getting Started
+---------------
+
+| Please use your best judgement on the type of chart, layout, and interpretation of the list below. This is an exercise to get hands-on experience with Tableau. This will be the first stop on a journey of many skills!
```

Comment:
```suggestion
Please use your best judgement on the type of chart, layout, and interpretation of the list below. This is an exercise to get hands-on experience with Tableau. This will be the first stop on a journey of many skills!
```